<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSliceControllerModule</class>
 <widget class="qSlicerWidget" name="qSlicerSliceControllerModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="ctkCollapsibleButton" name="SliceControllersCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Slice Controllers</string>
     </property>
     <layout class="QVBoxLayout" name="SliceControllersLayout"/>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="SliceInformationCollapsibleButton" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text" stdset="0">
      <string>Slice Information</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="SliceInformationLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Slice Node:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="MRMLSliceNodeComboBox" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Select a slice from the current scene</string>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLSliceNode</string>
           </stringlist>
          </property>
          <property name="showHidden" stdset="0">
           <bool>true</bool>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="qMRMLSliceInformationWidget" name="MRMLSliceInformationWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>226</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSliceInformationWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSliceInformationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerSliceControllerModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLSliceNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerSliceControllerModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLSliceInformationWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MRMLSliceNodeComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>MRMLSliceInformationWidget</receiver>
   <slot>setMRMLSliceNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
